src/datatypes/dialogTypes.ts

```diff
@@ -0,0 +1,30 @@
+import { brukertype } from '../types';
+
+export interface Dialog {
+    id: string;
+    overskrift: string;
+    aktivitetId?: string;
+    lest?: boolean;
+    sisteDato: string;
+    sisteTekst?: string;
+    erLestAvBruker?: boolean;
+    venterPaSvar?: boolean;
+    ferdigBehandlet?: boolean;
+    opprettetDato: string;
+    henvendelser?: Array<Henvendelse>;
```

Comment:
Jeg vet ikke helt hva du foretrekkker, men her kan man også skrive:
```suggestion
    henvendelser?: Henvendelse[];
```

Reply:
Enig med deg men det her er copy paste så tenkte ikke så mye over det :)